pkg/skaffold/config/options.go

```diff
@@ -50,6 +50,7 @@ type SkaffoldOptions struct {
 	AutoSync           bool
 	AutoDeploy         bool
 	RenderOnly         bool
+	Output             string
```

Comment:
I'd rename this here too: `RenderOutput`

Reply:
Fixed